test/integration/testdata/check.cni.golden

```diff
@@ -77,4 +77,8 @@ linkerd-control-plane-proxy
 √ control plane proxies are up-to-date
 √ control plane proxies and cli versions match
 
+linkerd-opaque-ports-definition
+-------------------------------
+√ the opaque-ports annotations have been properly defined
```

Comment:
TIOLI: we could say `opaque-ports are properly annotated` -- might sound a bit more Kubernetes like.

Reply:
Fine to me; I am far away from being a native speaker so I totally trust you on this :D

Changed!